Missionframework/stringtable.xml

```diff
@@ -6955,6 +6955,22 @@
             <Original>Arma 3 High Command</Original>
             <German>Arma 3 Oberbefehl</German>
         </Key>
+        <Key ID="STR_PARAM_SUPPMOD">
+            <Original>BI Support System access</Original>
+            <German>Zugriff auf BI Support System</German>
+        </Key>
+        <Key ID="STR_PARAM_SUPPMOD_COMMANDER">
+            <Original>Commander only</Original>
+            <German>Nur Kommandant</German>
+        </Key>
+        <Key ID="STR_PARAM_SUPPMOD_CMDRANDWHITELIST">
+            <Original>Commander and Whitelist only</Original>
+            <German>Nur Kommandant und Whitelist</German>
```

Comment:
I'm not sure if we need `command` and `commander and whitelist` params. Whitelist is empty by default so it's kinda the same as `commander` only by default.

Reply:
So removing the "Only Commander"?
Should be fine.